### ISSUE TITLE
eslint-disable-next-line being ignored

### ISSUE THREAD

**Issue by GarthDB** [codeclimate/codeclimate-eslint#127]
Line 83 is being flagged, but line 82 disables that check. It works locally, but not on codeclimate.
https://codeclimate.com/github/GarthDB/make-an-example/src/index.js

**Comment by pbrisbin** [comment]
Hi @GarthDB. It looks like that feature was added in ESLint 2.2.0, but our `stable` channel (which you got by default because you didn't specify one) runs ESLint 1.

Can you try with the following configuration?

```yaml
engines:
  eslint:
    enabled: true
    channel: eslint-2
```

More info can be found here: https://docs.codeclimate.com/docs/engine-channels

**Comment by GarthDB** [comment]
So good to know - I figured there was a missing piece.
